src/logger/constants.ts

```diff
@@ -0,0 +1,9 @@
+// AUTH ERRORS
+export const AUTH_GUARD_TOKEN_EXPIRED = 'AUTH_GUARD_TOKEN_EXPIRED';
+export const AUTH_GUARD_PARSING_ERROR = 'AUTH_GUARD_PARSING_ERROR';
+export const AUTH_GUARD_MISSING_HEADER = 'AUTH_GUARD_MISSING_HEADER';
+export const AUTH_GUARD_USER_NOT_FOUND = 'AUTH_GUARD_USER_NOT_FOUND';
+
+// ERROR LOGS
+export const UPDATE_FIREBASE_USER_EMAIL_ALREADY_EXISTS =
+  'UPDATE_FIREBASE_USER_EMAIL_ALREADY_EXISTS';
```

Comment:
there's an existing (utils/errors.ts)[./src/utils/errors.ts] file with FIREBASE_ERRORS, which aren't used for logging so maybe not one to combine, just flagging in case you have ideas